Jenkinsfile

```
@@ -1032,8 +1033,9 @@ pipeline {
             when {
                 beforeAgent true
                 // expression { skipTest != true }
+                expression { env.NO_CI_TESTING != 'true' }
                 expression {
-                    sh script: 'git show -s --format=%B | grep "^Skip-test: true"',
+                    sh script: 'git show -s --format=%B | grep "^Skip-test: true',
```

Looks like the closing double quote on the grep string got removed inadvertently:

```suggestion
                    sh script: 'git show -s --format=%B | grep "^Skip-test: true"',
```

---

Will fix.